slam-libraries/viam-cartographer/README.md

```diff
@@ -101,3 +98,7 @@ To run tests, after building, run the following
 ```bash
 ./scripts/test_cartographer.sh
 ```
+
```

Comment:
Where do we want to have documentation of current modes and future integrations with movement sensors? Here? Only Viam documentation? Both?

Reply:
I feel like having that stuff either in here or in the slam-libraries(or top level) readme would be fine, but would prob want to keep details limited to what you have now. Viam Documentation would contain more and should have more detail